Updated mkdocs.yml

callumbiggs opened the pull request:
Corrected spelling mistake for Reference sections, Iinterprocess Communication → Interprocess Communication

StephenTaylor-Kx commented:
We’ve resolved this now, haven’t we? If you agree, let’s scrap the PR.

sashkab commented:
@StephenTaylor-Kx -- I didn't check after submitting this PR. Looks like you did. Thanks, closing.

StephenTaylor-Kx commented:
Thanks for the catch. Are we there now?

Stephen



Stephen Taylor | Librarian | *Kx* | +44 7713 400852 | stephen@kx.com

On 25 July 2017 at 17:13, Aleks Bunin <notifications@github.com> wrote:

> @StephenTaylor-Kx <https://github.com/stephentaylor-kx> It seems you made
> the change in the repository, but did not deploy the change to the
> code.kx.com. Let's keep this open until you deploy. As a workaround we
> added redirect, but in the future, redirect might be removed and I'd like
> to avoid 404 errors.
>
> Thanks.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/KxSystems/docs/pull/9#issuecomment-317787575>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AT2hgAErDWnEeJh9Hctlbo5OFLGLH1jsks5sRhQ2gaJpZM4N7yp3>
> .
>

callumbiggs committed:
Updated mkdocs.yml

Corrected spelling mistake for Reference sections, Iinterprocess Communication → Interprocess Communication

ghost commented:
Already corrected, but thanks for your vigilance!
